src/Checkbox/Checkbox.component.tsx

```diff
@@ -79,6 +81,32 @@ const displayStyleRules: Function = (
   };
 };
 
+const indeterminateCheckboxRules: Function = (
+  cbSize: string,
+) => {
+  switch (cbSize) {
```

Comment:
This could be refactored to:
```
const ySize = { lg: 0, md: 1, sm: 2 }[cbSize];
const transform = `rotate(90deg) translateX(-1px) translateY(${ySize});`
return {
  'transform': transform,
  '-webkit-transform': transform,
  '-ms-transform': transform,
}; 
```